libmate-panel-applet/mate-panel-applet.h

```diff
@@ -123,6 +129,8 @@ int  mate_panel_applet_factory_setup_in_process (const gchar               *fact
 							  gpointer                   data);
 
 
+void mate_panel_applet_get_screen_geometry (GdkScreen *screen, int* width, int* height);
```

Comment:
Why this is added as public API? And why applet should care about screen size?

I would use:
```
void mate_panel_applet_get_monitor_geometry (MatePanelApplet *applet,
                                             int             *width,
                                             int             *height);
```

This way you can get correct monitor from applet.

Reply:
Removed from public API